IFC 4.3/Earthworks_01/readme.md

```diff
@@ -0,0 +1,50 @@
+
+## Earthworks\_01
+
+### Intent
+
+This unit test establishes a simple embankment of infrastructure facilities.
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+- ProjectSetup_1
+- Tin_1
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+
+- IfcEarthworksFill
+- IfcGeographicElement
+- IfcTriangulatedIrregularNetwork
+- IfcCartesianPointList3D
+- IfcEarthworksFillTypeEnum

```

Comment:
Is deprecated.

Reply:
IfcGeographicElement was replaced with IfcSolidStratum.